python/fapolicy_analyzer/ui/ancillary_trust_database_admin.py

```diff
@@ -156,3 +171,26 @@ def on_deployBtn_clicked(self, *args):
     def on_changeset_updated(self):
         deployBtn = self.get_object("deployBtn")
         deployBtn.set_sensitive(stateManager.is_dirty_queue())
+
+    def on_session_load(self, listPaPrs):
+        # flake doesn't like long lines so...
+        strFunction = "AncillaryTrustAdmin::on_session_load(): "
+        logging.debug(strFunction + "{} {}".format(len(listPaPrs), listPaPrs))
```

Comment:
You could try `logging.debug(f"AncillaryTrustAdmin::on_session_load(): {len(listPaPrs)} {listPaPrs}")` and see if that makes it under the max line size.

Reply:
Ok. Will consider that with the next PR, since it is a branch off of this one more or less.